app/models/post.rb

```diff
@@ -0,0 +1,22 @@
+class Post < ApplicationRecord
+  validates :title, presence: true
+  validates :text, presence: true
+  validates :comment_counter, presence: false
+  validates :likes_counter, presence: false
```

Comment:
- [ ] As per requirement, please implement a method that returns the 5 most recent comments for a given post.